src/main/java/org/broadinstitute/hellbender/utils/test/SamAssertionUtils.java

```diff
@@ -378,15 +378,25 @@ public static void assertCRAMContents(final File putativeCRAMFile) {
     private static void sortSam(final File input, final File output, final File reference, final ValidationStringency stringency) {
         final SortSam sort = new SortSam();
 
-        final ArgumentsBuilder args = new ArgumentsBuilder();
-
-        args.addInput(input);
-        args.addOutput(output);
+        // We can't use ArgumentsBuilder since it assumes GATK argument names, but we're running a Picard
```

Comment:
maybe we should change ArgumentBuilder to use the short names which I think are shared between picard and gatk?

Reply:
I can do so if you want, but addVCF will still do the wrong thing. I think it might be better to just avoid the confusion altogether.